src/Psalm/Issue/NonInvariantPropertyType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psalm\Issue;
+
+final class NonInvariantPropertyType extends CodeIssue
+{
+    public const ERROR_LEVEL = -1;
+    public const SHORTCODE = 235159;
```

Comment:
This is not very short. I'm not sure sure if there's a system for assigning shortcodes to issues.

Reply:
There's a script that outputs max used shortcode, so you could use that (on master) and `+1` the value it returns.